cli/pkg/kctrl/cmd/core/namespace_flags.go

```diff
@@ -60,6 +67,20 @@ func (s *NamespaceNameFlag) Resolve() error {
 	return nil
 }
 
+func (s *NamespaceFlags) CheckForDisallowedSharedNamespaces() error {
+	if s.AllowedSharedNamespaces {
+		return nil
+	}
+	for _, ns := range sharedNamespaces {
+		if s.Name == ns {
+			return fmt.Errorf(`Creating sensitive resources in a shared namespace (%s)
+			(hint: Specify a namespace using the '-n' flag or use kubeconfig to change default namespace 'kubectl config set-context --current --namespace=private-namespace'.
+			Or use '--dangerous-allow-allow-use-of-shared-namespace=%s' to allow use of shared namespace)`, s.Name, s.Name)
```

Comment:
is this error going to appear as multi-line indented to users? if, so that would be a little weird. if you want to split it for readability of code i would recommend using `"..." + "..." + "..."`

Reply:
Made the change